documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -4,6 +4,7 @@
 
 ### Resolved issues
 
+* (IDETECT-3657) Resolved an issue where intelligent persistent scans would fail if a project or version name included non-ASCII characters. 
```

Comment:
Looking at previous release comments, we tend to capitalize specific scan names. "... where Intelligent Persistent Scans would fail... ".  

@sig-cpotts What do you think?

Reply:
I would agree that has been the general practice, so we should also here if we have a scan type known as "Intelligent Persistent Scan".
This actually brings up a question, as we have "Intelligent" scan mode [detect.blackduck.scan.mode], but don't document it in the way we do Rapid, or Stateless etc. (I'll create a ticket to check on doing that.)

Reply:
You're right, I guess to prevent confusion we should just say "Intelligent Scan" and omit the "persistent"

Reply:
@sig-cpotts pushed a commit that capitalizes the scan name. I wasn't certain if I should remove the Persistent portion, just let me know if this is enough or if you'd like further updates. Thanks!

Reply:
Hey @dterrysynopsys that @sig-cpotts guy is sketchy. ; )
I think maybe do remove the "Persistent" just to avoid any confusion if somebody goes looking in the docs for a scan type under that exact name.
Thank you.